src/status_im/ui/screens/group/views.cljs

```diff
@@ -1,166 +1,172 @@
-     [view styles/settings-group-text-container
-      [text {:style styles/delete-group-text}
-       (label :t/delete-group)]
-      [text {:style styles/delete-group-prompt-text}
-       (label :t/delete-group-prompt)]]]]])
+     [react/view styles/settings-group-text-container
+      [react/text {:style styles/delete-group-text}
+       (i18n/label :t/delete-group)]
+      [react/text {:style styles/delete-group-prompt-text}
+       (i18n/label :t/delete-group-prompt)]]]]])
 
 (defn group-chat-settings-btns []
-  [view styles/settings-group-container
-   [view {:opacity 0.4}
-    [touchable-highlight {:on-press #()}
-     [view styles/settings-group-item
-      [view styles/settings-icon-container
+  [react/view styles/settings-group-container
+   [react/view {:opacity 0.4}
+    [react/touchable-highlight {:on-press #()}
+     [react/view styles/settings-group-item
+      [react/view styles/settings-icon-container
        [vi/icon :icons/speaker {:color :blue}]]
-      [view styles/settings-group-text-container
-       [text {:style styles/settings-group-text}
-        (label :t/mute-notifications)]]]]]
+      [react/view styles/settings-group-text-container
+       [react/text {:style styles/settings-group-text}
+        (i18n/label :t/mute-notifications)]]]]]
    [action-separator]
-   [action-button {:label     (label :t/clear-history)
+   [action-button {:label     (i18n/label :t/clear-history)
                    :icon      :icons/close
                    :icon-opts {:color :blue}
-                   :on-press  #(dispatch [:clear-history])}]
+                   :on-press  #(re-frame/dispatch [:clear-history])}]
    [action-separator]
-   [touchable-highlight {:on-press #(dispatch [:leave-group-chat])}
-    [view styles/settings-group-item
-     [view styles/delete-icon-container
+   [react/touchable-highlight {:on-press #(re-frame/dispatch [:leave-group-chat])}
+    [react/view styles/settings-group-item
+     [react/view styles/delete-icon-container
       [vi/icon :icons/arrow-right {:color :red}]]
-     [view styles/settings-group-text-container
-      [text {:style styles/delete-group-text}
-       (label :t/leave-chat)]]]]])
+     [react/view styles/settings-group-text-container
+      [react/text {:style styles/delete-group-text}
+       (i18n/label :t/leave-chat)]]]]])
 
 (defn more-btn [contacts-limit contacts-count on-press]
-  [view
+  [react/view
    [common/list-separator]
-   [view cstyles/show-all
-    [touchable-highlight {:on-press on-press}
-     [view
-      [text {:style cstyles/show-all-text
-             :uppercase? (get-in platform-specific [:uppercase?])
-             :font (if ios? :default :medium)}
-       (str (- contacts-count contacts-limit) " " (label :t/more))]]]]])
+   [react/view cstyles/show-all
+    [react/touchable-highlight {:on-press on-press}
+     [react/view
+      [react/text {:style cstyles/show-all-text
+                   :uppercase? (get-in platform-specific [:uppercase?])
```

Comment:
There is `uppercase?` fn in component.styles for this purpose

Reply:
Didn't touch that but good point!

Reply:
Done